docs/UserGuide.md

```diff
@@ -345,22 +321,120 @@ Examples:
+* Only the tag names are searched.<br>
+* Only full words will be matched. e.g. `Fri` will not match `Friday`.<br>
+* Employees whose tag(s) matching at least one keyword will be listed in the _Employee Directory_ pane
+ (i.e. `OR` search).  e.g. `Monday Friday` will find employees whose tags contain `Monday` or `Friday`.<br>
 
-Examples:
-* `c-tag-find friday` returns all employees whose tags contain `friday`.
-* `c-tag-find friday parttime` returns all employees whose tags contain `friday`, `parttime`.<br>
+<div markdown="span" class="alert alert-primary">
+
+**:information_source: Note:**<br>
+* In tCheck, if an employee contains a tag for a specific day, it means this employee is available on that day
+. You may set an employee's tags when adding the employee into tCheck (Refer to `c-add` command), or edit the
+tag(s) by editing employee's information feature (Refer to `c-edit` command).<br>
+
+</div>
+
+Example:
+* Let's say you want to find out who is available to work on `Monday` or `Friday`. In this case, the two 
+keywords are `Monday` and `Friday`. You can follow these steps.
+
+Steps to find employees who are available to work on `Monday` or `Friday`:
+1. Type `c-tag-find monday friday` into the _Command Box_.
+2. Press "Enter" to execute.
+
+Outcome:
+1. The _Result Display_ will show the total number of employees who are can work on `Monday` or `Friday`.
+
+2. Employees who are can work on `Monday` or `Friday` will be listed in the _Employee Directory_ pane.
+
+{Screenshot}
+
+#### 3.6 Locating all employees who are available to work today: `c-today`
+
+Finds employees whose tags contain today's day (i.e. `Wednesday`, `Thursday`, etc).
+
+Format: `c-today`
+
+* The search is case-insensitive (e.g `Friday` will match `friday`).<br>
+* The application will automatically get today's day based on the computer's current date.<br>
+* Only full words will be marched (e.g `Fri` will not match `Friday`).<br>
+
+<div markdown="span" class="alert alert-primary">
+
+**:information_source: Note:**<br>
+* In tCheck, if an employee contains a tag for a specific day, it means this employee is available on that day
+. You may set an employee's tags when adding the employee into tCheck (Refer to `c-add` command), or edit the
+tag(s) by editing employee's information feature (Refer to `c-edit` command).<br>
+
+</div>
+
+Example:
+* Let's say today is `Thursday`. You want to find out who are the available manpower for today. You can follow these
+ steps.
+
+Steps to find today's available manpower:
+1. Type `c-today` into the _Command Box_.
+2. Press "Enter" to execute.
+
+Outcome:
+1. The _Result Display_ will show 3 information:
+    
+    a. Today's day. (e.g. Today is THURSDAY.)
+    
+    b. Total number of employees who are available today.
+    
+    c. The success message.
+2. The available manpower for today will be listed in the _Employee Directory_ pane.
```

Comment:
Same issue here.

Reply:
It has been changed in the latest commit. Thanks!